roles/logstash/handlers/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+# handlers file for docker
+- name: restart zookeeper
```

Comment:
nope, needs to be zookeeper, as logstash isnt up when zookeeper starts so times out trying to send logs to logstash. Going to change it to use stdout/stderr so fits with the rest of the apps